packages/react-native/ReactAndroid/src/test/java/com/facebook/react/bridge/JavaScriptModuleRegistryTest.kt

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+package com.facebook.react.bridge
+
+import org.junit.Assert
+import org.junit.Test
+
+/**
+ * Tests for
+ *
+ * @see JavaScriptModuleRegistry
```

Comment:
NIT: can you use brackets for JavaScriptModuleRegistry:


```
/**
 * Tests for [JavaScriptModuleRegistry]
 * /
```

Reply:
Done ✅